src/solver/variable/economy/shortTermStorage.h

```diff
@@ -217,8 +235,24 @@ class ShortTermStorageEnergy
 
     void yearEnd(unsigned int year, unsigned int numSpace)
     {
-        VariableAccessorType::template ComputeStatistics<VCardType>(
-          pValuesForTheCurrentYear[numSpace]);
+        // Here we perform time-aggregations :
+        // ---------------------------------
+        // for a given MC year, from hourly results we compute daily, weekly, monthly and annual 
+        // results by aggregation operations (averages or sums).
+        // Caution : 
+        //  - level results are stored in columns of which indices satisfy : col_index % 3 == 1.
+        //    They are time-aggregated by means of averages
+        //  - injection and withdrawal results are stored in columns of which indices 
+        //    satisfy : col_index % 3 != 1.
+        //    They are time-aggregated by means of sums.
```

Comment:
You could make this "hack" a bit more expressive with booleans
```cpp
bool isLevel = (col_index % 3) == XXX;
bool isInjection = (col_index % 3) == XXX;
bool isWithdrawal = (col_index % 3) == XXX;
```

Reply:
Done. 
I corrected an error here (level columns satisfy `col_index % 3 == 2` instead of `col_index % 3 == 1`).